src/components/widgets/speed-direction-widget.tsx

```diff
@@ -0,0 +1,187 @@
+import * as React from "react";
+import { PureComponent } from "react";
+import styled from "styled-components";
+import { Icon } from "../icon";
+import WindWSDIcon from "../../assets/widget-icons/wind-speed-and-direction.svg";
+import WindSymbolOrangeIcon from "../../assets/widget-icons/wind-symbol-orange.svg";
+import WindSymbolBlueIcon from "../../assets/widget-icons/wind-symbol-blue.svg";
+import { HorizontalContainer, ValueContainer, ValueOutput, IconContainer } from "../styled-containers";
+import { WidgetPanelTypes, kWidgetPanelInfo, round } from "../../utilities/widget";
+
+interface ValueDividerProps {
+  backgroundColor?: string;
+}
+const ValueDivider = styled.div`
+  width: 1px;
+  height: 21px;
+  margin: 0 5px 0 5px;
+  background-color: ${(p: ValueDividerProps) => `${p.backgroundColor ? `${p.backgroundColor}` : "#FFDBAC"}`};
```

Comment:
```suggestion
  background-color: ${(p: ValueDividerProps) => `${p.backgroundColor || "#FFDBAC"}`};
```
or `??` depending on how you feel about other falsy values.

Reply:
Fixed.